spec/ics-003-connection-semantics/README.md

```diff
@@ -353,7 +354,9 @@ function connOpenAck(
   consensusHeight: Height) {
     abortTransactionUnless(consensusHeight < getCurrentHeight())
     connection = provableStore.get(connectionPath(identifier))
-    abortTransactionUnless(connection.state === INIT || connection.state === TRYOPEN)
+    abortTransactionUnless(
+        (connection.state === INIT && connection.version.indexOf(version) !== -1)
```

Comment:
it can't both be equal and `indexOf`, I think you want `indexOf`

Reply:
Not sure I understand your comment.

This conditions is trying to ensure that: If the (current) state of the connection is `INIT`, then the incoming argument `version` must exist inside the list of versions `connection.version`. Agree this is not the ideal way to code this check.

Reply:
Oh, never mind, just got confused reading the diff